src/hooks/analytics/useTrackAppOpened.ts

```diff
@@ -1,26 +1,25 @@
-import { usePostHog } from 'posthog-react-native';
 import React from 'react';
 import { AppState, NativeEventSubscription } from 'react-native';
-
-const APPLICATION_OPENED_EVENT = 'Application Opened';
+import useTrackEvent from 'hooks/analytics/useTrackEvent';
+import { Events } from 'types/analytics';
 
 /**
  * Hook to track when the user opens the applications.
  */
 const useTrackAppOpened = () => {
+  const trackEvent = useTrackEvent();
   const [appStarted, setAppStarted] = React.useState(false);
-  const postHog = usePostHog();
 
   React.useEffect(() => {
     let subscription: NativeEventSubscription | undefined;
-    if (!appStarted && postHog) {
+    if (!appStarted) {
       if (AppState.currentState === 'active') {
-        postHog.capture(APPLICATION_OPENED_EVENT);
+        trackEvent(Events.ApplicationOpened);
         setAppStarted(true);
       } else {
         subscription = AppState.addEventListener('change', (state) => {
           if (state === 'active') {
-            postHog.capture(APPLICATION_OPENED_EVENT);
+            trackEvent(Events.ApplicationOpened);
```

Comment:
Here as well we should update `appStarted` after the event `ApplicationOpened` has been captured correctly.